shared/common-adapters/avatar.render.desktop.js

```diff
@@ -106,7 +106,7 @@ class AvatarRender extends React.PureComponent<Props, State> {
 
     return (
       <div
-        onClick={this.props.onClick}
+        onClick={this.props.onEditAvatarClick ? this.props.onEditAvatarClick : this.props.onClick}
```

Comment:
you can just contorl what onClick means in the connector. this can just be this.props.onclick